pyclowder/datasets.py

```diff
@@ -14,6 +14,126 @@
 from pyclowder.collections import get_datasets, get_child_collections, delete as delete_collection
 from pyclowder.utils import StatusMessage
 
+def create_empty_dataset(host, clowder_user, clowder_pass, datasetname, description, parentid=None, spaceid=None):
```

Comment:
this seems to be the same as [create_empty](https://github.com/clowder-framework/pyclowder/blob/cf74e3e1624b6d63fbae4a89c8c0b2251e6478d3/pyclowder/datasets.py#L138) (quick glance). Can we add the username/password/key option to that function and have them all be default None?